README.md

```diff
@@ -1,32 +1,29 @@
 # Auto-Voice-Channels
 
-A Discord bot that automatically creates voice channels as they are needed.
+### A Discord bot that automatically creates voice channels as they are needed.
 
-Public bot invite link: <https://discordapp.com/api/oauth2/authorize?client_id=479393422705426432&permissions=286280784&scope=bot>
+Public bot invite link: <https://discordapp.com/api/oauth2/authorize?client_id=479393422705426432&permissions=286280784&scope=bot>  
+Beta bot invite link: <https://discordapp.com/api/oauth2/authorize?client_id=675405085752164372&permissions=286280784&scope=bot>  
+Discord server: <https://discord.gg/HT6GNhJ>  
+Patreon: <https://www.patreon.com/pixaal>  
 
-Requires:
+## Requires:
 
-* Python 3.5.3+
-* pytz
-* psutil
+* [Python 3.5.3+](https://www.python.org/downloads/)
+* [discord.py](https://pypi.org/project/discord.py/) `pip install discord.py`
+* [pytz](https://pypi.org/project/pytz/) `pip install discord.py`
+* [psutil](https://pypi.org/project/psutil/) `pip install psutil`
 
 ## Quick start:
 
-* Clone: `git clone git@github.com:gregzaal/Auto-Voice-Channels.git`
-* `cd Auto-Voice-Channels`
+* Clone the repository: `git clone git@github.com:gregzaal/Auto-Voice-Channels.git`
+* Go to the directory: `cd Auto-Voice-Channels`
 * Make folder to store guild settings: `mkdir guilds`
 * Install pip: `sudo apt-get -y install python3-pip`
 * Install venv: `pip3 install virtualenv`
 * Make venv: `python3 -m virtualenv bot-env`
 * Use venv: `. bot-env/bin/activate`
 * Install dependencies:
-
-```
-python3 -m pip install -U discord.py
```

Comment:
Would say to remove the `python3 -m ` if youre gonna do it for the other modules, that would also make it windows compatible aswell.

Reply:
https://github.com/gregzaal/Auto-Voice-Channels/pull/8/commits/1bceb178e4beffb7a05360a0022832140a6c89c3